### ISSUE TITLE
Replace Mailgun with Amazon SES

### ISSUE THREAD

**Issue by TomK32** [a-song-a-day/a-song-a-day#21]
Amazon SES is far cheaper than Mailgun. The question is what features from Mailgun are actually used and needed; and wether SES does offer the same or similar features.

[Intro to SES](http://docs.aws.amazon.com/ses/latest/DeveloperGuide/Welcome.html)

**Comment by shannonbyrne** [comment]
Yeah, we talked about that a bit when we built the existing backend but decided on Mailgun - I forget why though. I think there are also cheaper options within Mailgun.

**Comment by TomK32** [comment]
Are you using any things on the mailgun dashboard?
Cost-wise, SES will be 20% of mailgun according to the [calculator](http://calculator.s3.amazonaws.com/index.html) (Amazon SES on the left sidebar).

**Comment by shannonbyrne** [comment]
We've been using it to track email sends, opens, clicks which is v. important especially moving forward. Not sure if AWS offers the same thing? Also looping in @ilikescience (our technical advisor) because he's been super involved on this side of things.

**Comment by ilikescience** [comment]
👋 hi, sorry I missed this tag, I've had GH notifications off for a while because Planetary hired more devs and I got booted from most dev work :)

I think SES is a smart choice; my money's always on Amazon for reliability and stability.

Most of ASAD's email needs are in tracking opens and clicks in a structured way — Mailgun has a pretty nice API for pulling events out of the stream. I know SES has similar functionality, but I'm less familiar with it.

Overall, I say go for it!
